osm.py

```diff
@@ -97,6 +97,18 @@ def find_obsidian_default_root():
         CONFIG_DIRECTORIES_TRIED.append(potential_dir)
 
 
+def load_items_to_copy():
+
+    items_to_copy_contents = safe_read_contents(COPY_LIST_FILE, '')
+
+    for line in items_to_copy_contents.splitlines():
+        if not line:
+            continue  # Ignore blank lines
+        line = line.strip()  # Allow indentation, esp. for comments.
```

Comment:
Yeah, this strip() should be above the `not line`, that's fixed in a later commit.